sw/device/lib/testing/aon_timer_testutils.c

```diff
@@ -67,3 +68,17 @@ void aon_timer_testutils_watchdog_config(const dif_aon_timer_t *aon_timer,
   CHECK_DIF_OK(dif_aon_timer_watchdog_start(aon_timer, bark_cycles, bite_cycles,
                                             pause_in_sleep, false));
 }
+
+void aon_timer_testutils_shutdown(const dif_aon_timer_t *aon_timer) {
+  CHECK_DIF_OK(dif_aon_timer_wakeup_stop(aon_timer));
+  CHECK_DIF_OK(dif_aon_timer_watchdog_stop(aon_timer));
+  CHECK_DIF_OK(dif_aon_timer_clear_wakeup_cause(aon_timer));
+  CHECK_DIF_OK(dif_aon_timer_irq_acknowledge_all(aon_timer));
+  // Read and verify both timers are actually disabled. This ensures the
+  // synchronization from the core clock to the AON clock domain completed.
+  bool enabled;
```

Comment:
i know there's a `DifToggleEnabled` type, not sure if the software folks have a preference for that here or if `bool` is okay.  I think this is fine.

Reply:
IMO `bool` is fine for non DIF code.

Reply:
yeah i think so too.